migration-engine/migration-engine-tests/tests/create_migration/create_migration_tests.rs

```diff
@@ -347,7 +347,7 @@ fn create_enum_step_only_rendered_when_needed(api: TestApi) {
                             "id" INTEGER NOT NULL,
                             "mood" "Mood" NOT NULL,
 
-                            PRIMARY KEY ("id")
+                            CONSTRAINT "Cat_pkey" PRIMARY KEY ("id")
```

Comment:
?

Reply:
That's the one thing I'm worried about. I'll double check that before merging.

Reply:
Ok I double checked, and it's just that the test is terrible (using MySQL datamodel connector with postgres database, and it working somehow). I'll have a look at these tests in a bit more detail, but I don't think anything is broken apart from the tests.

Reply:
I investigated more. 99% sure it's just the test, primary key constraint names are tested in other places and work as expected (and didn't change) there.